sdk/Cargo.toml

```diff
@@ -32,6 +32,7 @@ prefix-hex = { version = "0.7.0", default-features = false, features = [ "primit
 primitive-types = { version = "0.12.1", default-features = false }
 serde = { version = "1.0.160", default-features = false, features = [ "derive" ] }
 serde_json = { version = "1.0.95", default-features = false, features = [ "alloc" ] }
+time = { version = "0.3.20", default-features = false, features = [ "serde", "macros" ] }
```

Comment:
Really mandatory?

Reply:
No, not mandatory. It's nice to be able to define the date but we could just use a string.

Reply:
I mean, is the dep optional depending on features?

Reply:
Ah, yes, indeed.